.github/workflows/test-wheel-linux.yml

```diff
@@ -180,6 +180,26 @@ jobs:
           host-platform: ${{ inputs.host-platform }}
           cuda-version: ${{ inputs.cuda-version }}
 
+      - name: Set up compute-sanitizer
+        run: |
+          # We don't test compute-sanitizer on CTK<12 because backporting fixes is too much effort
+          # We only test compute-sanitizer on python 3.12 arbitrarily; we don't need to use sanitizer on the entire matrix
+          # Only local ctk installs have compute-sanitizer; there is not wheel for it
+          if [[ "${{ inputs.python-version }}" == "3.12" && "${{ inputs.cuda-version }}" != "11.8.0" && "${{ inputs.local-ctk }}" == 1 ]]; then
+            COMPUTE_SANITIZER="${CUDA_HOME}/bin/compute-sanitizer"
+            COMPUTE_SANITIZER_VERSION=$(${COMPUTE_SANITIZER} --version | grep -Eo "[0-9]{4}\.[0-9]\.[0-9]" | sed -e 's/\.//g')
+            SANITIZER_CMD="${COMPUTE_SANITIZER} --target-processes=all --launch-timeout=0 --tool=memcheck --error-exitcode=1"
+            if [[ "$COMPUTE_SANITIZER_VERSION" -ge 202111 ]]; then
+                SANITIZER_CMD="${SANITIZER_CMD} --padding=32"
+            fi
+            echo "CUDA_PYTHON_SANITIZER_RUNNING=1" >> $GITHUB_ENV
```

Comment:
Do we really need the `CUDA_PYTHON_SANITIZER_RUNNING` variable?

I see below you're doing this:
```
echo "COMPUTE_SANITIZER_VERSION=${COMPUTE_SANITIZER_VERSION}" >> $GITHUB_ENV
```

Could we just query that in the tests? E.g. for most cases:

```
@pytest.mark.skipif(
    os.environ.get("COMPUTE_SANITIZER_VERSION") is not None,
    reason="The compute-sanitizer is running, and this test intentionally causes an API error.",
)
```

Maybe later we'll have cases that want to look at the version number, then we wouldn't need anything new or special.

Reply:
I've just removed `COMPUTE_SANITIZER_VERSION`. It's better to have tests controlled by a `CUDA_PYTHON_` specific variable, and we shouldn't need to adjust the tests according to the compute sanitizer version. Instead, we should just always used the latest version of compute-sanitizer.